test_site_fixtures/test_website_content.json

```diff
@@ -0,0 +1,1718 @@
+[
+  {
+    "model": "websites.websitecontent",
+    "pk": 522807,
+    "fields": {
+      "deleted": null,
+      "deleted_by_cascade": false,
+      "created_on": "2023-10-19T22:18:28.810Z",
+      "updated_on": "2023-11-16T15:46:22.867Z",
+      "owner": null,
+      "updated_by": null,
+      "website": "554fc3fa-2f4b-4a94-87f4-19e51ae36920",
+      "text_id": "5388b0c3-c599-4c42-8a3c-f89beed7c154",
+      "title": "example_jpg.jpg",
+      "type": "resource",
+      "parent": null,
+      "markdown": "",
+      "metadata": {
+        "body": "",
+        "draft": false,
+        "license": "https://creativecommons.org/licenses/by-nc-sa/4.0/",
+        "file_size": 154826,
+        "file_type": "image/jpeg",
+        "video_files": {
+          "archive_url": "",
+          "video_captions_file": "",
+          "video_thumbnail_file": "",
+          "video_transcript_file": ""
+        },
+        "content_type": "resource",
+        "resourcetype": "Image",
+        "image_metadata": { "credit": "", "caption": "", "image-alt": "" },
+        "video_metadata": {
+          "video_tags": "",
+          "youtube_id": "",
+          "video_speakers": "",
+          "youtube_description": ""
+        },
+        "learning_resource_types": ["Activity Assignments"]
+      },
+      "is_page_content": true,
+      "filename": "example_jpg",
+      "dirpath": "content/resources",
+      "file": "courses/123-ocw-ci-test-course-fall-2022/example_jpg.jpg"
```

Comment:
These need to be updated. https://github.com/mitodl/ocw-hugo-themes/pull/1280 changes expected urls from `.../123-ocw-ci-test-course-fall-2022/...` to `.../ocw-ci-test-course/...`.

The content generated from these fixtures fails tests.